README.md

```diff
@@ -9,31 +9,155 @@ is [Disnake](https://github.com/DisnakeDev/disnake/).
 Did you ever want to experience the information overload and stress of a phone operator?
 Well, then you're at the right place!
 
-We've created a first stressful Discord game, in which you play a phone operator for a restaurant.
+We've created the first stressful Discord game, in which you play a phone operator for multiple restaurants.
```

Comment:
That's a little too much

Reply:
Could you eloborate?

Reply:
We probably didn't create the _first_ stressful Discord game.

Reply:
Or did you mean that we created the first stressful Discord game _where you are a phone operator for multiple restaurants_? Then it's probably fine.

Reply:
Thats what i meant. The sentence is also structured in such way that this interpretation makes gramaticly sense.
I ofc cannot say with 100% certainty that its actually the first, but its extremely likely.

Also, a bit of "clickbait" wont hurt 😋.

Reply:
👌